BeerProject/spring-boot-ehcache/cache/file/breweryCache_795dcad2f65c78cb9c83b773a4bf591c28f7885a/offheap-disk-store/ehcache-disk-store.meta

```diff
@@ -0,0 +1,4 @@
+#Key and value types
+#Wed Sep 09 01:16:14 CEST 2020
```

Comment:
jw.

Reply:
ale gdzie to jest? Ten plik ehcache.xml został usuniety. Tam mogły byc takie rzeczy